scripts/collision_detection_example.py

```diff
@@ -46,6 +46,21 @@ def load_environment(client_id):
     }
     return bodies
 
+def create_user_debug_params(gui_id):
+    return {
+        'collision_margin': pyb.addUserDebugParameter('collision_margin', 0, 0.2, 0.01, physicsClientId=gui_id),
+        'lbr_iiwa_joint1': pyb.addUserDebugParameter('lbr_iiwa_joint1', -2*np.pi, 2*np.pi, 0, physicsClientId=gui_id),
```

Comment:
Could you populate the joint entries here with a loop to make it a bit cleaner?

Reply:
Fixed